src/scss/includes/panels/categories.scss

```diff
@@ -6,47 +6,48 @@
 
 .category__panel__pj {
   width: 100%;
-  height: 50px;
-  padding: 0 20px;
-  display: flex;
-  line-height: 18px;
-  align-items: center;
-  justify-content: space-between;
-
-  &_title {
-    font-weight: normal;
-    color: $primary_text;
-    background: url('../images/pagesjaunes.svg') no-repeat left center;
-    background-size: 18px;
-    padding-left: 24px;
-  }
-
-  &_partnership {
-    display: inline-flex;
-    align-items: center;
-    font-size: 12px;
-    color: $secondary_text;
-
-    .icon-icon_info {
-      font-size: 16px;
-      margin-right: 4px;
-    }
-  }
+  height: 36px;
+  position: absolute;
+  bottom: 0;
+  border-top: 1px solid $grey-light;
+  background: $background;
+  line-height: 36px;
+  text-align: center;
+  font-size: 14px;
+  color: $grey-semi-darkness;
+  border-radius: 0 0 12px 12px;
 }
 
 .category__panel__items {
   animation: appear 600ms forwards;
 }
 
 .category__panel__error {
-  padding: 16px 16px 32px 16px
+  padding: 16px 16px 32px 16px;
+
+  & ~ .category__panel__pj {
+    display: none;
+  }
 }
 
-@media (max-width: 640px) {
-  .category__panel__pj {
-    height: 20px;
+@media (min-width: 641px) {
+  .category__panel {
+
+    &.panel--pj {
+      .panel-content {
+        overflow: hidden;
+      }
+
+      .category__panel__items {
+        height: calc(100vh - 170px);
```

Comment:
Here you reintroduce a hardcoded computation with some "magic" value, that we made disappeared for the panel content. Maybe it's needed, but I'm not sure. What was the problem with the flex-based layout?

Reply:
When I try to do it similarly to panel-header and panel-content, with the proper columns and shrink properties, the items container doesn't scroll anymore... 🤔

Reply:
I tried the flex-based layout in the inspector, it worked:
```css
.category__panel.panel--pj .panel-content {
   overflow: hidden;
   display: flex;
   flex-direction: column;
}
.category__panel.panel--pj .category__panel__items {
   overflow-y: auto;
}
.category__panel__pj {
   /* position: absolute; to be removed */
   flex-shrink: 0;
}
```

Reply:
yes IIRC we tested this

Reply:
thanks for the tip! I was close, now it works :)